src/pages/HomePage.tsx

```diff
@@ -73,6 +118,10 @@ const HomePage = () => {
         <br />
         {`Personal Setting: ${JSON.stringify(personalSetting)}`}
         <br />
+        {/* githubContentRef.currentをそのまま文字列として出力するとインデントが崩れるのでtextareaのプロパティとして文字列を表示 */}
+        <p style={{ margin: 0 }}>github code: </p>
+        <textarea value={githubContentRef.current} />
```

Comment:
警告が出てました。

```react-dom.development.js:67 Warning: You provided a `value` prop to a form field without an `onChange` handler. This will render a read-only field. If the field should be mutable use `defaultValue`. Otherwise, set either `onChange` or `readOnly`.```

Reply:
textarea タグの value プロパティ代入で起こっていたエラーを消すため
textarea タグ-> pre タグに変更しました